package.json

```diff
@@ -47,6 +47,7 @@
   },
   "dependencies": {
     "chartjs-color": "^2.0.0",
-    "moment": "^2.10.6"
+    "moment": "^2.10.6",
+    "element-resize-detector": "^1.1.10"
```

Comment:
We're using this library for other purposes and haven't noticed any issues with it. It supports two ways of detecting resize events (object-based and scroll-based, see the [ERD repository](https://github.com/wnr/element-resize-detector), and this [PDF](http://arxiv.org/pdf/1511.01223v1.pdf) for more information) I like the fact that it's a battle tested way of detecting resize events, but I also understand why you would like to avoid having another dependency.